subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/CustomPluginIntegrationTest.groovy

```diff
@@ -138,10 +138,6 @@ class CustomPlugin implements Plugin<Project> {
         project.ext.custom = 'value'
     }
 }
-        """
```

Comment:
I think we can actually ditch this test entirely because we have coverage for this same thing in `GradleImplDepsVisibilityIntegrationTest`